install_config/router/f5_router.adoc

```diff
@@ -68,14 +64,16 @@ ifdef::openshift-origin[]
 * Ensure you have xref:../../install_config/router/index.adoc#creating-the-router-service-account[created the router service account].
 endif::[]
 
-{product-title} supports only the following *F5 BIG-IP®* versions:
+The F5 router plug-in for {product-title} supports only the following *F5 BIG-IP* versions:
 
 * 11.x
 * 12.x
 
+The F5 BIG-IP Controller for OpenShift supports the versions found in this matrix on Clouddocs.f5.com. see link:https://clouddocs.f5.com/containers/v2/releases_and_versioning.html[F5 BIG-IP Controller for OpenShift releases and versioning]
```

Comment:
s/versions/{product-title} versions
s/in this matrix on Clouddocs.f5.com. see link:https://clouddocs.f5.com/containers/v2/releases_and_versioning.html[F5 BIG-IP Controller for OpenShift releases and versioning]
/in the link:https://clouddocs.f5.com/containers/v2/releases_and_versioning.html[Releases and Versioning] page in the F5 documentation.

That page doesn't list OCP 3.10 in the compatibility matrix. Please update your docs if you want us to put this in the 3.10 collection.

Reply:
Agree with the changes to the text. 
As for 3.10 support on our matrix: that is being worked now but the testing has not been completed.